backend/src/resolvers/code.resolver.ts

```diff
@@ -12,4 +14,40 @@ export default class CodeResolver {
 
 		return codes;
 	}
+
+	// find a code for a projectid
```

Comment:
```suggestion
	@Arg([UserRole.VISITOR, UserRole.ADMIN])
```
controle utilisateur

Reply:
Ce n'est pas plutôt @Authorized à la place de @Arg?

Reply:
Yep my bad, j'ai glissé chef ;p